Content.Shared/Anomaly/Effects/Components/EntitySpawnAnomalyComponent.cs

```diff
@@ -36,8 +33,21 @@ public sealed partial class EntitySpawnAnomalyComponent : Component
     public float SpawnRange = 5f;
 
     /// <summary>
-    /// The tile that is spawned by the anomaly's effect
+    /// Whether or not anomaly spawns entities on Pulse
     /// </summary>
-    [DataField("floorTileId", customTypeSerializer: typeof(PrototypeIdSerializer<ContentTileDefinition>)), ViewVariables(VVAccess.ReadWrite)]
-    public string FloorTileId = "FloorFlesh";
+    [DataField("spawnOnPulse"), ViewVariables(VVAccess.ReadWrite)]
```

Comment:
```suggestion
    [DataField, ViewVariables(VVAccess.ReadWrite)]
```

datafield names are inferred if left blank. juts remove it and it should work